blog/_posts/2017-01-24-cudanative.md

```diff
@@ -0,0 +1,247 @@
+---
+layout: post
+title:  "Technical preview: Native GPU programming with CUDAnative.jl"
+author: <a href="https://github.com/maleadt">Tim Besard</a>
+cudanative_tree: https://github.com/JuliaGPU/CUDAnative.jl/blob/6e6ec23ef4def367dec6ae0ba3e43f1d9daae7ec
+---
+
+After 2 years of slow but steady development, we would like to announce the first preview
+release of native GPU programming capabilities for Julia. You can now write your CUDA
+kernels in Julia, albeit with some restrictions, making it possible to use Julia's
+high-level language features to write high-performance GPU code.
+
+The programming support we're demonstrating here today consists of the low-level building
+blocks, sitting at the same abstraction level of CUDA C. You should be interested if you
+know (or want to learn) how to program a parallel accelerator like a GPU, while dealing with
```

Comment:
`, like a GPU, in native Julia, while taking advantage of low-level communication semantics.`